### PR TITLE
Feature/l10n cleanup

### PR DESCRIPTION
**Pull request by sknorr**
@tomschr Could you look at this?

I just need to work out with varkoly whether the Hungarian book name formatting we generate is good enough...

### CODE REVIEW

File: suse2013/common/l10n/it.xml [hunk 1]
```diff
@@ -70,52 +66,61 @@
...
+      <l:template name="appendix" style="nonumber" text="Appendice «%t»"/>
+      <l:template name="bridgehead" style="nonumber" text="sezione «%t»"/>
+      <l:template name="chapter" style="nonumber" text="capitolo «%t»"/>
+      <l:template name="equation" style="nonumber" text="Equazione «%t»"/>
+      <l:template name="example" style="nonumber" text="Esempio «%t»"/>
+      <l:template name="figure" style="nonumber" text="figura «%t»"/>
+      <l:template name="part" style="nonumber" text="Parte «%t»"/>
+      <l:template name="procedure" style="nonumber" text="Procedura «%t»"/>
+      <l:template name="productionset" style="nonumber" text="Produzione «%t»"/>
+      <l:template name="qandadiv" style="nonumber" text="D e R «%t»"/>
```
Comment (tomschr) [inline suse2013/common/l10n/it.xml]: Is this text ("D e R «%t»") correct?

Comment (sknorr) [inline suse2013/common/l10n/it.xml]: I only changed the quotes here ... see line 106 above.

File: suse2013/fo/xref.xsl [hunk 2]
```diff
@@ -44,7 +44,7 @@
                  external-destination="{$ulink.url}">
     <xsl:choose>
       <xsl:when test="count(child::node()) = 0 or
-                      (string(.) = $url) or
+                      normalize-space(string(.) = $url) or
```
Comment (tomschr) [inline suse2013/fo/xref.xsl]: This looks a bit strange to me. I think, you want to write this:

```
 normalize-space(string(.)) = $url or
```

You can even remove the `string()` function as `normalize-space(.)` does it automatically.

**Comment by tomschr** [comment]
LGTM :+1:

**Comment by sknorr** [comment]
Okay, will then fix the remaining two things (Hungarian & node(.))...
and then will merge myself...

**Comment by sknorr** [comment]
Node is fixed, Hungarian still needs review...

**Pull request merged**